Gemfile

```diff
@@ -22,6 +22,7 @@ group :development, :test do
   gem "pry"
   gem "pry-stack_explorer"
   gem "pry-byebug"
+  gem "solidus_dev_support", github: "nvandoorn/solidus_dev_support"
```

Comment:
I took a brief look at the changes you have here and I am curious if this is something we can override in our extensions' spec helper while we PR this upstream 🤔